airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/retriers/http_response_filter.py

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Set
+
+import requests
+from airbyte_cdk.sources.declarative.interpolation.interpolated_boolean import InterpolatedBoolean
+from airbyte_cdk.sources.streams.http.http import HttpStream
+
+
+class HttpResponseFilter:
+    TOO_MANY_REQUESTS_ERRORS = set([429])
```

Comment:
if a user wanted to ignore 429s (because an API throws 404 to mean rate limit, for example), would that happen by configuring this class to catch 404s, and the retrier to ignore 429s? asking since I was wondering if there's any merit to allowing the user to configure the HTTP statuses used here

Reply:
added that example: https://github.com/airbytehq/airbyte/pull/14330/files#diff-acecabdc247ed135ba4cb18a7865401227e2cc17d2de4164afe3762df933415aR60-R67